plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiFileStatus.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.hudi;
+
+import io.trino.filesystem.FileEntry;
+import org.apache.hadoop.fs.Path;
+
+import java.util.Objects;
+
+import static com.google.common.base.MoreObjects.toStringHelper;
+import static java.util.Objects.requireNonNull;
+
+public class HudiFileStatus
+        implements Comparable<HudiFileStatus>
```

Comment:
I didn't see any places where this object is used as a `Comparable`. It might make sense to use a custom `Comparator` by doing `Comparator.comparing(HudiFileStatus::path)` in the place where it's needed, since the ordering seems like it would be specific to the usage (though hard to tell since I didn't see where it's used).

Reply:
Removed. It's not really needed.